packages/design-system/src/components/N8nTooltip/Tooltip.vue

```diff
@@ -48,15 +37,15 @@ export default defineComponent({
 		<slot />
 		<template #content>
 			<slot name="content">
-				<div v-html="content"></div>
+				<div v-html="props.content"></div>
```

Comment:
You should omit `props` in `<template>`.

```suggestion
				<div v-html="content"></div>
```

Reply:
If I remove the `props` it complains about the properties not existing:

![image](https://github.com/user-attachments/assets/5865459b-739e-4d79-9676-1d4cf2fd57b5)

It seems extending the `ElTooltip.props` messes up the props. Any suggestion what's the correct way to extend another components props with the composition API?